openflow15/group.go

```diff
@@ -162,7 +162,7 @@ func (g *GroupMod) UnmarshalBinary(data []byte) (err error) {
 	g.CommandBucketId = binary.BigEndian.Uint32(data[n:])
 	n += 4
 
-	for n < g.Header.Length {
+	for n < g.BucketArrayLen+24 {
```

Comment:
this seems unrelated to the rest of the PR?

Reply:
Properties (including experimenter ones) are encoded after buckets. But because of `n < g.Header.Length` here we're trying to decode property as a bucket and fail.